integration-test/src/main/java/com/sequenceiq/it/cloudbreak/newway/testcase/mock/LdapClusterTest.java

```diff
@@ -40,41 +38,60 @@ public void tearDown(Object[] data) {
     }
 
     @Test(dataProvider = TEST_CONTEXT_WITH_MOCK)
+    @Description(
+            given = "a valid cluster request with ldap configuration",
+            when = "calling create cluster",
+            then = "the ldap should be configured on the cluster")
     public void testCreateClusterWithLdap(MockedTestContext testContext) {
         testContext.getModel().getAmbariMock().postSyncLdap();
         testContext.getModel().getAmbariMock().putConfigureLdap();
         testContext.given(LdapConfigTestDto.class)
                 .when(ldapConfigTestClient.post())
                 .given(AmbariRepositoryV4Entity.class)
-                .given(AmbariEntity.class).withAmbariRepoDetails()
-                .given(ClusterEntity.class).withLdapConfig().withAmbari()
-                .given(StackTestDto.class).withCluster()
+                .given(AmbariEntity.class)
+                .withAmbariRepoDetails()
+                .given(ClusterEntity.class)
+                .withLdapConfig()
+                .withAmbari()
+                .given(StackTestDto.class)
+                .withCluster()
                 .when(Stack.postV4())
                 .await(STACK_AVAILABLE)
-                .then(MockVerification.verify(HttpMethod.POST, AmbariMock.LDAP_SYNC_EVENTS))
-                .then(MockVerification.verify(HttpMethod.PUT, AmbariMock.LDAP_CONFIGURATION))
+                // TODO should be checked
```

Comment:
It should be checked or not? A Follow up jira would be better than a `TODO` in a comment